react.md

```diff
@@ -5,39 +5,52 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Correctable True/False: Mark the following TRUE or FALSE. If FALSE, correct the statement to be TRUE.
 
-- React is a modern, efficient answer to complex UI applications (true/false)
-- React will only render on the server using Node.js (true/false)
-- React is a full stack framework for modern web applications (true/false)
-- React is a flexible library that plays the role of V in an MVC framework (true/false)
-- You should always update a component's state directly using this.state (true/false)
-- React is made up of encapsulated components that manage their own state (true/false)
-- React components render HTML (true/false)
+- React is a modern, efficient answer to complex UI applications (true/false) true
+- React will only render on the server using Node.js (true/false) true
```

Comment:
React is pretty flexible. It can be rendered on the server with lots of different languages.